### ISSUE TITLE
Not openning trades, a bit of help, thanks

### ISSUE THREAD

**Issue by carboce1** [iterativv/NostalgiaForInfinity#350]
I dont know why are not openning trades. It has been an entire day.

The configuration is the one recommended by the library for Binance. No errors on the LOG, using Docker.

![image](https://github.com/iterativv/NostalgiaForInfinity/assets/91984530/91b6ebe8-9edb-4745-8ef4-40a5359707be)

**Comment by iterativv** [comment]
There were trades at 22, at least.

Are you certain that the CPU is not too slow ? If not, then change the volume lists to 100 and 75 pairs.

**Comment by maddoxdev** [comment]
Is Raspberry PI 4 (4gb) boosted to 2 GHz not enough?

**Comment by maddoxdev** [comment]
> There were trades at 22, at least.
> 
> Are you certain that the CPU is not too slow ? If not, then change the volume lists to 100 and 75 pairs.

Is Raspberry PI 4 (4gb) boosted to 2 GHz not enough?

**Comment by verhovensky** [comment]
Same for me, no trades open for 3 days on dry run

**Comment by ne0x86** [comment]
same form me too, no trades open in >48 hours

**Comment by m-kubik** [comment]
anyone getting trades? I have nothing for two days

**Comment by GeorgesAlkhouri** [comment]
Same

**Comment by FoxCraft67** [comment]
Same here

**Comment by godshades** [comment]
NIF have many protections rule, so this will enter least trade than other strategies

**Comment by m-kubik** [comment]
> NIF has many protection rules, so this will involve fewer trades than other strategies.

Indeed, it seems quite puzzling when starting to use it for the first time. I wasn't sure everything was working. However, after a few days, some trades have been executed. There aren't many of them, but it's quite impressive so far; each trade has ended in profit.

**Comment by FoxCraft67** [comment]
Ok so I'll wait a little longer, thanks for the reply.

**Comment by AlexiaChen** [comment]
> > NIF has many protection rules, so this will involve fewer trades than other strategies.
> 
> Indeed, it seems quite puzzling when starting to use it for the first time. I wasn't sure everything was working. However, after a few days, some trades have been executed. There aren't many of them, but it's quite impressive so far; each trade has ended in profit.

Keep looking

**Comment by AlexiaChen** [comment]
> > > NIF has many protection rules, so this will involve fewer trades than other strategies.
> > 
> > Indeed, it seems quite puzzling when starting to use it for the first time. I wasn't sure everything was working. However, after a few days, some trades have been executed. There aren't many of them, but it's quite impressive so far; each trade has ended in profit.
> 
> there are no trades opening on my freqtrade UI for 3 hours, will keep looking

updated .  received opening trade. just wait and be patient

**Comment by alfirin** [comment]
Same here no trade.

**Comment by iterativv** [comment]
This issue should be resolved. Don't hesitate to open a new one, thank you !
